tests/container/component_invalid_app2/Fail2Handler.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.test;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.yahoo.jdisc.Request;
+import com.yahoo.jdisc.Response;
+import com.yahoo.jdisc.handler.*;
+
+public class Fail2Handler extends AbstractRequestHandler {
+
+    public Fail2Handler() {
+        String nullString = null;
+        System.out.println("This will throw an IndexOutOfBoundsException: " + nullString.length());
```

Comment:
I would expect a `NullPointerException`

Reply:
Just checking the quality of our code reviews 😉